RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -4008,7 +4035,7 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				CODE_SIGN_STYLE = Automatic;
 				INFOPLIST_FILE = Tests/BackendIntegrationTests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
This allows us to have properties inside of `ObserverModeManager` without having to do `Any` hacks.
We never run integration tests below iOS 16 so I think it's time in the interest of keeping these tests as maintainable as possible.